SPECS/expat/expat.spec

```diff
@@ -55,17 +53,22 @@ rm -rf %{buildroot}/*
 %license COPYING
 %doc AUTHORS Changes
 %{_bindir}/*
-%{_mandir}/man1/*
 
 %files devel
 %{_includedir}/*
 %{_libdir}/pkgconfig/*
 %{_libdir}/libexpat.so
+%{_libdir}/cmake/expat-%{version}
 
 %files libs
 %{_libdir}/libexpat.so.*
```

Comment:
Can we add the major sover number to this line (e.g. if the library is `libexpat.so.X.Y.Z` and we have symlink `libexpat.so.X`, this line should be changed to `%{_libdir}/libexpat.so.X*`)? This helps us be more cognizant of when we have probable breaking ABI/API changes that require us to bump and rebuild dependent packages.

Reply:
Fixed, thanks for pointing that out.